librz/core/cmd/cmd_debug.c

```diff
@@ -40,9 +40,6 @@ static const char *help_msg_d[] = {
 	"dk", "[?]", "List, send, get, set, signal handlers of child",
 	"dL", "[?]", "List or set debugger handler",
 	"dm", "[?]", "Show memory maps",
-	"do", "[?]", "Open process (reload, alias for 'oo')",
```

Comment:
this needs to remain, probably with a slightly different description though.

The reason is that `d` is still not converted to rzshell, so these strings are used to provide the help for the sub-commands of `d`, `do` included.